packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -114,8 +112,8 @@ - (FlutterError *)flutterError {
       @"type" : type,
       @"document" : documentChange.document.data,
       @"path" : documentChange.document.reference.path,
-      @"oldIndex" : [NSNumber numberWithInt:documentChange.oldIndex],
-      @"newIndex" : [NSNumber numberWithInt:documentChange.newIndex],
+      @"oldIndex" : [NSNumber numberWithUnsignedInteger:documentChange.oldIndex],
+      @"newIndex" : [NSNumber numberWithUnsignedInteger:documentChange.newIndex],
```

Comment:
AFAIK oldIndex and newIndex can be set to `-1` (or at least it was before). Did this change recently?

Reply:
Not sure, the property is declared as `NSUInteger` in latest

Reply:
@ahmedb just to double check what do you refer to as "latest"? E.g. the exact version and maybe even link to the source?

Reply:
Hi,

I am using flutter v0.8.2 (https://github.com/flutter/flutter/releases)

Xcode 9.4.1 on Mac OS X 10.13

And the master branch of this repository ( https://github.com/flutter/plugins )

Reply:
Thanks, I was mostly interested in versions of official pod libraries of Firebase SDK. I’m on latest 5.8.0 and arrayContains is definitely in place and working.